dissect/target/plugins/os/windows/network.py

```diff
@@ -301,9 +319,8 @@ def _interfaces(self) -> Iterator[WindowsInterfaceRecord]:
                     if not conf or not any(
                         [
                             conf["dns"],
```

Comment:
For the sake of consistency, can you order these the same as the record definition?

Reply:
This one is still pending.